transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannel.java

```diff
@@ -31,6 +35,7 @@
 
     private final EpollServerSocketChannelConfig config;
     private volatile InetSocketAddress local;
+    private volatile Set<InetAddress> tcpMd5Sig = Collections.emptySet();
```

Comment:
Could this field and the `tcpMd5Sig()` method have a better name? It does not contains any signature but a set of addresses only.
 Add a line note